src/app/report-rollup/report-summary-graphs/report-summary-pie-chart/report-summary-pie-chart.component.ts

```diff
@@ -109,5 +115,68 @@ export class ReportSummaryPieChartComponent implements OnInit {
     Plotly.newPlot(this.reportSummaryPieChart.nativeElement, data, layout, modebarBtns);
   }
 
+  drawPrintPlot() {
+    let valuesArr: Array<number>;
+    let textTemplate: string;
+    if (this.dataOption == 'energy') {
+      valuesArr = this.pieChartData.map(dataItem => {
+        return dataItem.energyUsed
+      });
+      textTemplate = '%{label}:<br>%{value:,.0f} ' + this.energyUnit;
+    }
+    else if (this.dataOption == 'cost') {
+      valuesArr = this.pieChartData.map(dataItem => {
+        return dataItem.annualCost
+      });
+      textTemplate = '%{label}:<br>%{value:$,.0f}';
+    }
+    else if (this.dataOption == 'energySavings') {
+      valuesArr = this.pieChartData.map(dataItem => {
+        return dataItem.energySavings
+      });
+      textTemplate = '%{label}:<br>%{value:$,.0f}' + this.energyUnit;
+    }
+    else if (this.dataOption == 'costSavings') {
+      valuesArr = this.pieChartData.map(dataItem => {
+        return dataItem.costSavings
+      });
+      textTemplate = '%{label}:<br>%{value:$,.0f}';
+    }
+    else if (this.dataOption == 'carbon') {
+      valuesArr = this.pieChartData.map(dataItem => {
+        return dataItem.carbonEmissions
+      });
+      textTemplate = '%{label}:<br>%{value:,.0f} ' + 'tonne CO<sub>2</sub>';
+    }
+    var data = [{
```

Comment:
Change 'var's to 'let'. The project should always use let for block scoped variables. Most projects will use const as well.